CMakeLists.txt

```diff
@@ -45,7 +45,7 @@ if("${HALIDE_SYSTEM_LIBS}" STREQUAL "")
   # It's theoretically possible that this could be legitimately empty,
```

Comment:
This comment sounds kinda stale now.

Reply:
Yes, it probably should be edited, I changed the message in the warning slightly, but I don't know much about other llvm versions so I decided not to expand on this issue. Should I change the comment?